app/views/provider/admin/dashboards/_service_navigation.html.slim

```diff
@@ -43,7 +44,7 @@ nav.DashboardNavigation
                                                  admin_service_end_user_plans_path(service)
 
     // Integration
-    - if can?(:edit, service) && !service.has_traffic?
+    - if can?(:manage, :plans) && !service.has_traffic?
```

Comment:
Are you sure? It sounds more related to editing the service than managing plans 🤔 But if you are sure, I believe you

Reply:
<img width="793" alt="screenshot 2018-10-03 18 51 03" src="https://user-images.githubusercontent.com/54224/46425982-7c786e80-c73d-11e8-97d0-b92e9bbd846d.png">

the plans permission also does the integration

Reply:
aaaah cool 👍 thanks 😄